tests/functional/src/main/java/org/mule/functional/api/classloading/isolation/ArtifactIsolatedClassLoaderBuilder.java

```diff
@@ -166,6 +169,18 @@ public ArtifactIsolatedClassLoaderBuilder setExportClasses(final Set<Class> expo
   }
 
   /**
+   * Sets the {@link List} of {@link String}s to define the Maven artifactIds that should be ignore no matter if they are
```

Comment:
It will not ignore the dependencies, what it will be ignored is to register the artifact as service, but the artifact and its dependencies could be included in another service (case that we talked the other day) or if that is not the case it will be added to the container (including its dependencies too).
It is not clear yet how to avoid this second scenario to me yet. Same would happen with extensions if they are not discovered.
This happens because the container urls are defined by subtracting from whole classpath the ones classified for app, plugins and services.